rest/src/main/java/org/jboss/pnc/rest/endpoints/internal/DeliverableAnalysisEndpointImpl.java

```diff
@@ -38,9 +47,27 @@ public class DeliverableAnalysisEndpointImpl implements DeliverableAnalysisEndpo
     @Inject
     private DeliverableAnalyzerOperationMapper deliverableAnalyzerOperationMapper;
 
+    @Inject
+    private ManagedExecutorService executorService;
+
     @Override
     public void completeAnalysis(AnalysisResult response) {
-        resultProcessor.completeAnalysis(transformToModelAnalysisResult(response));
+        executorService.execute(() -> {
```

Comment:
please add `@RespondWithStatus(Response.Status.ACCEPTED)` to the interface method, so it returns 202 ACCEPTED instad of 204. And update the swagger doc acordingly.

Reply:
It's supposed to go on the inerface